.github/workflows/wheel_macos_arm64.yml

```diff
@@ -0,0 +1,64 @@
+name: Wheels
+on:
+  push:
+    branches:
+      - master
+
+env:
+  CIBW_BUILD: 'cp37-* cp38-* cp39-*'
+
+  # MacOS specific build settings
+  CIBW_BEFORE_ALL_MACOS: |
+    brew uninstall --force oclint
+    brew install gcc libomp
+
+  # Python build settings
+  CIBW_BEFORE_BUILD: |
+    pip install pybind11
+
+  # Testing of built wheels
+  CIBW_TEST_REQUIRES: numpy scipy pytest pytest-cov pytest-mock flaky
+
+  # Note: breaking up tests so that no test cases hang on Windows, to be reverted
+  CIBW_TEST_COMMAND: |
+    pip install numpy==1.19.5 scipy
+    pl-device-test --device=lightning.qubit --skip-ops -x --tb=short --no-flaky-report
+
+jobs:
+  mac-wheels-arm64:
+    strategy:
```

Comment:
Do we need an if statement here?

Reply:
This should only run on merging with master (top lines of the file).

Reply:
I have also cleaned up the if statements on the other files, as you were right earlier, they were not essential.